lib/compat/wordpress-6.1/class-wp-theme-json-resolver-6-1.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * WP_Theme_JSON_Resolver_6_1 class
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that abstracts the processing of the different data sources
+ * for site-level config and offers an API to work with them.
+ *
+ * This class is for internal core usage and is not supposed to be used by extenders (plugins and/or themes).
+ * This is a low-level API that may need to do breaking changes. Please,
+ * use get_global_settings, get_global_styles, and get_global_stylesheet instead.
+ *
+ * @access private
+ */
+class WP_Theme_JSON_Resolver_6_1 extends WP_Theme_JSON_Resolver_6_0 {
```

Comment:
Have the methods of this class changed from `6.0`? They look the same to me at a quick glance — but I imagine it's needed here so that `__DIR__` points to the correct `theme.json` or `theme-i18n.json` file? (Just want to make sure I understand how it's hooked together 😀)

Reply:
Yeh, needed here in order to pick up the new theme.json files in the 6.1 folder  - I think this is the correct way to handle this.

Reply:
Thanks for confirming, looks right to me! I'll resolve this comment 🙂